docs/platform/howto/use-billing-groups.rst

```diff
@@ -1,63 +1,29 @@
-Manage billing groups in the Aiven Console
-==========================================
-
-Billing groups enable you to set a common billing profile for multiple
-projects and generate a consolidated invoice. To know more about the key benefits read the :doc:`related documentation <../concepts/billing-groups>`.
-
-Billing groups and Aiven accounts
----------------------------------
-
-An Aiven **Account** is the entry point for creating and managing billing groups. The following are options available for billing groups via the `Aiven Console <https://console.aiven.io/>`_:
-
-- Within an Account you can create one or more billing groups, depending on the need
-- You can group projects under an Account into one or more billing groups
-- You can set a **primary billing group** for an Account. All future projects associated with this Account are then added to the primary billing group by default.
-- You can move projects under an Account from one billing group to another.
-
-Aiven credits are available for all projects associated with the Account. You can choose to apply the credits to a specific project within the Account.
-
 Manage billing groups
----------------------
-
-To view and manage billing groups in the `Aiven Console <https://console.aiven.io/>`_:
-
-#. Click the current project and select **See all projects & accounts**.
+=====================
 
-#. Click an account that you want to manage and then click the **Billing** tab.
+View and manage billing groups in the Aiven Console
```

Comment:
I'd reword the doc title into "Manage billing groups in the Aiven Console".
Also, I'd reword the first section title into "View billing groups" since there's noting about managing and it's already stated that's done in the console (see the new doc title).